src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -261,14 +346,20 @@ private void move(Vector spdVec, double rot, boolean rateLimit) {
     m_rearRight.setDesiredState(swerveModuleStates[3]);
   }
 
+  /**
+   * applies slewrate limiting to the given control vector
```

Comment:
If time we should get slew rate limiting working again.

Reply:
What do you mean "working again"? What's not working about it?

Reply:
When we comment out slew rate code, robot moves according to joystick mappings, otherwise it just stalls. I'm going to look at it in more depth tonight so we can get it reimplemented, since it's a necessary safety precaution.

Reply:
I would have actually expected this to show up earlier, but Vecor.rot() actually rotated in the wrong direction (since it was originally made for screens, where the y-axis is flipped) which is fixed now, and hopefully should work, but it still needs tested